rest_framework/throttling.py

```diff
@@ -120,7 +121,10 @@ def allow_request(self, request, view):
         if self.key is None:
             return True
 
-        self.history = self.cache.get(self.key, [])
+        self.history = self.cache.get(self.key)
```

Comment:
Does there need to be a some sort of check that the returned value from the cache is really a `deque` instance? If the cache is populated by a previous version of DRF `self.history` will still be a `list` instance right?